src/components/Timer/Timer.js

```diff
@@ -0,0 +1,32 @@
+import './Timer.css';
+import TextTimer from '../TextTimer/TextTimer';
+import LightsaberTimer from '../LightsaberTimer/LightsaberTimer';
+
+export default function Timer(callback = () => null) {
+    let minute = 1;
+    let sec = 30;
+    let allSeconds = (minute * 60 + sec);
```

Comment:
add this as helper function

`convertTimeToSeconds = (minutes, seconds) => minutes * 60 + seconds`

and then it may be ``` let remainingSeconds = Math.round(((convertTimeToSeconds(min, sec)) / allSeconds * 100), 2);```